### PR TITLE
Add past bookings to seed

### PR DESCRIPTION
**Pull request by johnckealy**

### CODE REVIEW

File: db/seeds.rb [hunk 1]
```diff
@@ -62,17 +62,30 @@
...
+200.times do
 
-100.times do
+  m = rand(1..7)
+  sd = rand(1..15)
+  ed = sd + rand(1..15)
+  st = Date.new(2020,m,sd)
+  ed = Date.new(2020,m,ed)
+
+  user_this_booking = users.sample.id
```
Comment (kathrin298) [inline db/seeds.rb]: wouldn't use the id but the user directly, id's might change if you reseed your file without dropping the database. instead go for users.sample and then down below in the booking, user: user_this_booking

File: db/seeds.rb [hunk 2]
```diff
@@ -62,17 +62,30 @@
...
+  duration = (ed - st).to_i
+  booking.total_fee = (animal.daily_rate * duration).round(2)
 
   review = Review.new(title: Faker::GreekPhilosophers.quote,
-                         content: Faker::TvShows::RickAndMorty.quote,
-                         rating: rand(5),
-                         user_id: rand(4))
+                      content: Faker::TvShows::RickAndMorty.quote,
+                      rating: rand(2..5),
+                      user_id: user_this_booking)
```
Comment (kathrin298) [inline db/seeds.rb]: same here, assign it as user object not user id

File: db/seeds.rb [hunk 3]
```diff
@@ -62,17 +62,30 @@
...
-                           delivery_included: true, user_id: rand(4), start_date: Date.new(2020,4,5),
-                           animal_id: rand(9), end_date: Date.new(2020,4,6), total_fee: rand(500))
+                        delivery_included: true, user_id: user_this_booking, start_date: st,
+                        end_date: ed)
+  booking.animal = animal
+
+  duration = (ed - st).to_i
+  booking.total_fee = (animal.daily_rate * duration).round(2)
 
   review = Review.new(title: Faker::GreekPhilosophers.quote,
```
Comment (kathrin298) [inline db/seeds.rb]: can we add more than one review to each booking?

**Comment by johnckealy** [comment]
requested changes are now pushed

**Pull request merged**